apps/token/src/config/links.ts

```diff
@@ -10,4 +10,5 @@ export const Links = {
   DISCORD: 'https://vega.xyz/discord',
   STAKING_GUIDE:
     'https://docs.vega.xyz/docs/mainnet/concepts/vega-chain/#staking-on-vega',
+  GOVERNANCE_PAGE: 'https://vega.xyz/governance',
```

Comment:
question (blocking): does this need to change based on environement? Is there a governance place we should be linking to for fairground. cc @candida-d who would know

Reply:
The Vega website link doesn't need to change per environment. I don't think there's a plan to have a different page for different environs